src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+appendRow.addEventListener('click', () => {
+  const cloneRow = tbody.lastElementChild.cloneNode(true);
+
+  tbody.append(cloneRow);
+
+  if (tbody.children.length > 9) {
```

Comment:
just move all magic numbers into constants and it will be the great solution👍

Reply:
done